docs/conf.py

```diff
@@ -59,17 +59,17 @@
 # You can specify multiple suffix as a list of string:
 #
 source_suffix = {
-        ".rst": "restructuredtext",
-        ".txt": "markdown",
-        ".md": "markdown",
+    ".rst": "restructuredtext",
+    ".txt": "markdown",
+    ".md": "markdown",
 }
 
 # The master toctree document.
 master_doc = "index"
 
 # General information about the project.
 project = u"aicsimageio"
-copyright = u'2019, Allen Institute for Cell Science'
+copyright = u"2019, Allen Institute for Cell Science"
```

Comment:
2020

Reply:
I'm like 99% sure I fixed this in the the 4.0 proposed API branch but will fix it here as well haha